roles/network_plugin/flannel/templates/docker-network-options-systemd

```diff
@@ -0,0 +1,2 @@
+[Service]
+Environment="DOCKER_NETWORK_OPTIONS=--bip={{ flannel_subnet }} --mtu={{ flannel_mtu }}"
```

Comment:
Yes, the value of `DOCKER_NETWORK_OPTIONS` will be used across all supported OSes. For CoreOS we were using systemd drop-ins exclusively to set options so I continued that here. For the other OSes we were setting `DOCKER_NETWORK_OPTIONS` with `EnvironmentFile`s though they were possibly just copied out of [this example](https://docs.docker.com/engine/admin/systemd/#/custom-docker-daemon-options).

If upstart (Ubuntu 14.04 and 14.10) support is not a concern, I could change this to use drop-ins across all OSes, which would be simpler and much more consistent with recommended systemd usage. Upstart is broken in other ways right now anyway. Thoughts?